### PR TITLE
Release 2.7.0

### PR DESCRIPTION
**Pull request by tuomashatakka**
:)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,46 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+### React Components
+
+#### Added
+
+- [Labels and Tags] New tag variants with rounded corners
+
+#### Changed
+
+- [Input fields] Input fields' buttons have larger clickable area around them
```
Comment (harriplappalainen) [inline CHANGELOG.md:27]: Should we put the affected components straight in here, like `[DateInput, SearchInput, PasswordInput]` ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,46 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+#### Added
+
+- [Icons] New icon UI/Navigation/swap-user
+- [Labels and Tags] New tag variants with rounded corners
+
+### React Components
+
+#### Added
+
+- [Labels and Tags] New tag variants with rounded corners
```
Comment (minevala) [inline CHANGELOG.md:23]: We could use [Tags] if the change only affected tags.

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,46 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+#### Fixed
+
+- [Theme] Fix useTheme hook
+- [FileInput] Fix an issue with FileInput crashing
+
+### Core
+
+#### Added
+
+- [Labels and Tags] New tag variants with rounded corners
```
Comment (minevala) [inline CHANGELOG.md:38]: We could use [Tags] if the change only affected tags

**Review by minevala — COMMENTED** [review]
We could also remove the release list from what-is-new page because it is not up-to-date anymore

File: CHANGELOG.md (hunk 1 above)
Comment (tuomashatakka) [reply CHANGELOG.md]: Yes

**Review by minevala — APPROVED** [review]
Nice 👍 
Remember to update the release draft as well

**Pull request merged**